pages.es/common/d2.md

```diff
@@ -0,0 +1,28 @@
+# d2
+
+> Un lenguaje moderno de scripting de diagramas que convierte texto en diagramas.
+> Más información: <https://d2lang.com/tour/man>.
+
+- Compila y renderiza un archivo fuente D2 en un archivo SVG o PNG:
```

Comment:
The description of the first example was recently updated from:
`- Compile and render a D2 source file into an SVG or PNG file:`
 to:
`- Compile and render a D2 source file into an output file:`.
This should be updated there too.

Reply:
Done